src/main/java/com/breskul/bring/annotations/Bean.java

```diff
@@ -0,0 +1,17 @@
+package com.breskul.bring.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Indicates that a method produces a bean which will be stored and managed in Bring container.
+ *
+ */
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Bean {
```

Comment:
`Bean` should have a `value` method to store the name of the bean.

Reply:
Done.